app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PetApplicationsController < ApplicationController
+  
+  def create
+    pet_application = PetApplication.new({application_id: params[:application_id], pet_id: params[:pet_id]})
+    if pet_application.save
+      redirect_to "/applications/#{params[:application_id]}"
+    else
+      redirect_to "/applications/#{params[:application_id]}"
+      flash[:alert] = "Error: #{error_message(application.errors)}"
+    end
+  end
+end
```

Comment:
For US 11/12, you would probably eventually end up changing the status of a PetApplication record based on when an individual pet is approved. This logic could be done in this controller in an `update` action